### PR TITLE
Fix generator for blacklight 5.10

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

**Comment by mjgiarlo** [comment]
Travis is :unamused:

**Comment by jcoyne** [comment]
@mjgiarlo yeah, I know. I believe it's due to blacklight_advanced_search doing some nasty monkey patching.

**Comment by awead** [comment]
This uses blacklight_advanced_search from master to avoid lots of deprecation warnings. if we want, we can wait and merge this once new versions of that and hydra-head have been released,

**Comment by jcoyne** [comment]
@awead thanks for tackling this, but I'd say wait until versions of hydra-head and blacklight_advanced_search are out.

**Comment by awead** [comment]
@jcoyne agreed. Do you want to cut another release of hydra-head? I can ask the Blacklighters about the advanced search gem.

File: spec/views/batch/edit.html.erb_spec.rb [hunk 1]
```diff
@@ -1,8 +1,12 @@
 require 'spec_helper'
 
 describe 'batch/edit.html.erb' do
-  let(:batch) { stub_model(Batch, id: '123') }
-  let(:generic_file) { stub_model(GenericFile, id: nil, depositor: 'bob', rights: ['']) }
+  let(:batch) { Batch.create }
+  let(:generic_file) do
+    GenericFile.new(title: ['some title']).tap do |f|
```
Comment (mjgiarlo) [inline spec/views/batch/edit.html.erb_spec.rb]: Are you intentionally not saving this file?

Comment (jcoyne) [inline spec/views/batch/edit.html.erb_spec.rb]: the tap is unnecessary here. AF::Base.new takes a block in AF 9

Comment (awead) [inline spec/views/batch/edit.html.erb_spec.rb]: @mjgiarlo it doesn't need to be save. We just need to have an actual batch in hand for the test.

Comment (mjgiarlo) [inline spec/views/batch/edit.html.erb_spec.rb]: :ok:

**Pull request merged**